client/components/notice/index.tsx

```diff
@@ -76,6 +78,8 @@ export default function Notice( {
 	status,
 	text,
 	theme = 'dark',
+	ariaLive,
+	role,
```

Comment:
```suggestion
	role = 'status',
```

Nit: In my opinion this is easier to read and follows the same pattern as the other props with a default value in this component (like `theme`)

Reply:
Done in 147ac4ca44242a45c0ec2f6fd84b30bca0505bbb